tests/Checker/Preparations/Use_Minimal_Theme_Preparation_Tests.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Tests for the Check_Context class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Checker\Preparation;
+use WordPress\Plugin_Check\Checker\Preparations\Use_Minimal_Theme_Preparation;
+
+class Use_Minimal_Theme_Preparation_Tests extends WP_UnitTestCase {
+	public function set_up() {
+		parent::set_up();
+
+		$this->theme_slug = 'wp-empty-theme';
+		$this->theme_dir  = WP_PLUGIN_DIR . '/' . basename( TESTS_PLUGIN_DIR ) . '/test-content/themes';
+
+		$this->preparation = new Use_Minimal_Theme_Preparation( $this->theme_slug, $this->theme_dir );
+
+		$this->cleanup = $this->preparation->prepare();
```

Comment:
We should have a test for what this method does, rather than calling it in `set_up`.